### ISSUE TITLE
Functions tips sometimes are wrong sized (height)

### ISSUE THREAD

**Issue by zaynyatyi** [fdorg/flashdevelop#959]
Sometimes functions tips are too small by height
![screenshot 2015-11-24 12 40 39](https://cloud.githubusercontent.com/assets/327729/11364372/bdbe5854-92a8-11e5-843f-398e7e3d848b.png)
Not always and height is always different if it's not same size as description content.

**Comment by Neverbirth** [comment]
Judging by your previous report, I'd say this is only with CrossOver?

**Comment by zaynyatyi** [comment]
Yes, crossover issue only. Sorry, i forgot to mention that.

**Comment by Meychi** [comment]
I think this is due to the internal tooltip control we use, it sometimes doesn't the update the size correctly. I need to look if this could be fixed somehow.

**Comment by Neverbirth** [comment]
I'm curious. Could you try the "Rewritten completion list and call tip" build from here?:

http://www.flashdevelop.org/community/viewtopic.php?f=9&t=12472

**Comment by Meychi** [comment]
@Neverbirth: the same problems persist with that build.

**Comment by Neverbirth** [comment]
Really? Quite interesting, because the tip is inside a different container, and sizing logic is not 100% the same if memory serves me right (but maybe I found a way to have it back so the changes with the current one were fewer).

**Comment by Meychi** [comment]
Yep. The font of the tooltip shows larger on first time and on the next time slightly smaller and then the container fits correctly. Sometimes it shows an empty container as described above. Maybe there are multiple issues with it.

**Comment by Neverbirth** [comment]
> The font of the tooltip shows larger on first time and on the next time slightly smaller and then the container fits correctly

Where? The official build, my branch or both? Just curious.

**Comment by Meychi** [comment]
Its been like that for sometime already in all builds.

**Comment by elsassph** [comment]
That is definitely a Wine-specific issue - that's the only case I've seen it. There is something broken in the measurement of the rich-text field when running with Wine.

**Comment by Neverbirth** [comment]
It may also be related to how Wine handles some resizing stuff, because the window relies on some automatic resizing logic, maybe if the resizing would be done in a more manual way capturing the rich text field resize events and then resize the container it would work as expected.

**Comment by Neverbirth** [comment]
Oh, sorry, I remembered wrong, the sizing is not automatic, it's based on some manual calculations.
